lib/one_model.ex

```diff
@@ -161,6 +161,9 @@ defmodule OneModel do
       * `select: atom | list`
         * Passing an atom will return the value of the given atom key
         * Passing a list of keys will return the default schema with the values for the specified keys.
+      * `select_map: atom | list`
+        * Passing an atom will return the value of the given atom key
```

Comment:
I don't think this makes sense. If I want to get back the single value I can just pass `select: key`, so why not let this do some thing different from that, which is to be consistent with the rest of the select_map function and have it return `%{key: value}`.  
Either that or don't accept an atom and, but let me have a single entry list, which gets the same `{%key: value}` response I said above.

Reply:
I don't agree that this option (if kept and my proposal is rejected) should allow for atoms to be passed. It contradicts the purpose and name of the option to "select a map".

Reply:
Ok. I'll raise if only an atom is given.  Returing a map with 1 key seems silly, just pass [:key] for that.

Unless anyone has a better suggestion.